sksurv/tree/tree.py

```diff
@@ -229,6 +236,11 @@ def fit(self, X, y, sample_weight=None, check_input=True):
         # one "class" for CHF, one for survival function
         self.n_classes_ = np.ones(self.n_outputs_, dtype=np.intp) * 2
 
+        if self.low_memory:
+            self.n_outputs_ = 1
+            # one "class" for the sum over the CHF, one for the sum over the survival function
+            self.n_classes_ = np.ones(self.n_outputs_, dtype=np.intp) * 2
```

Comment:
Since you only need a single output (the sum over the CHF), it should be sufficient to set `n_outputs_` to 1 and don't set `self.n_classes_` at all.

Reply:
Just noting that not setting `self.n_classes_` or setting it to `1` gave an error, hence only removed the ` * 2` to give `self.n_classes_ = np.ones(self.n_outputs_, dtype=np.intp)` now then for a single output.